io.github.Fndroid.clash_for_windows.appdata.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>io.github.Fndroid.clash_for_windows</id>
+  <launchable type="desktop-id">io.github.Fndroid.clash_for_windows.desktop</launchable>
+  <name>Clash for Windows</name>
+  <developer_name>Fndroid</developer_name>
+  <summary>Clash's graphical client, based on Electron.</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary=https://github.com/Fndroid/clash_for_windows_pkg/issues/153</project_license>
+  <url type="homepage">https://github.com/Fndroid/clash_for_windows_pkg</url>
+  <description>
+    <p>Clash is a rules-based, cross-platform proxy application core program written in Go. Clash for Windows is a graphical Clash branch available on Linux. The Clash API is used to configure and control the Clash core application for easy visualization and use by users.</p>
```

Comment:
I still have no idea what this does.

Reply:
In countries and regions where the Internet is heavily censored, sites such as Google, Github, etc. are inaccessible. This application is used to solve this problem. For example, I am now able to access Github because I depend on this application. So for users in censored areas, this app is useful and it is unspoken that this app is useful. But in areas where there is free access to the Internet, this app is arguably useless.

Reply:
And to protect myself, I can't describe the usefulness of this application in too much detail. I hope you can understand.